libs/core/execution/tests/unit/algorithm_run_loop.cpp

```diff
@@ -761,23 +760,6 @@ void test_future_sender()
         HPX_TEST_EQ(f.get(), 3);
     }
 
-    // Make a future from a sender
-    {
-        ex::run_loop loop;
-        auto sched = loop.get_scheduler();
-
-        auto s = ex::schedule(sched) | ex::then([]() { return 3; });
-        auto f = ex::make_future(std::move(s));
-        HPX_TEST_EQ(f.get(), 3);
-    }
-    // Make a future from a scheduler
-    {
-        ex::run_loop loop;
-        auto sched = loop.get_scheduler();
-
-        auto f = ex::make_future(sched);
-    }
-
```

Comment:
Why did you remove these tests?

Reply:
I added those tests at some point but I think because I was trying to understand how `make_future` works.
I removed them because the first one is pretty much a duplicate of  https://github.com/STEllAR-GROUP/hpx/pull/6431/commits/ef31254b19a6b730ca8863df58b9b95c3e79e1ec#diff-b11a40f80e11363af9d5469224fdb4e94487ee5ff99c30c7d4219a5ab40c85d8L799 and the second one is using `make_future` on a scheduler instead of a sender which is not correct in the first place.